go.mod

```diff
@@ -132,10 +249,20 @@ require (
 	k8s.io/mount-utils v0.0.0
 	k8s.io/pod-security-admission v0.0.0
 	k8s.io/sample-apiserver v0.0.0
-	k8s.io/system-validators v1.5.0
-	k8s.io/utils v0.0.0-20210707171843-4b05e18ac7d9
-	sigs.k8s.io/structured-merge-diff/v4 v4.1.2
-	sigs.k8s.io/yaml v1.2.0
+	sigs.k8s.io/apiserver-network-proxy/konnectivity-client v0.0.22 // indirect
+	sigs.k8s.io/kustomize/api v0.8.11 // indirect
+	sigs.k8s.io/kustomize/kustomize/v4 v4.2.0 // indirect
+	sigs.k8s.io/kustomize/kyaml v0.11.0 // indirect
+)
+
+require (
```

Comment:
the three require blocks is interesting... I think it's related to https://tip.golang.org/doc/go1.17#go-command:

> Because the number of explicit requirements may be substantially larger in an expanded Go 1.17 go.mod file, the newly-added requirements on indirect dependencies in a go 1.17 module are maintained in a separate require block from the block containing direct dependencies.

but the end result here is... weird... there aren't cleanly separated "direct requires" and "indirect requires" blocks... I'm pulling this branch down and trying to see what's going on